azure-pipelines.yml

```diff
@@ -160,14 +160,28 @@ jobs:
     inputs:
       secureFile: 'NORDIC_SIGNING_CERTIFICATE.pfx'
     condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'win'))
-    displayName: 'Donwload certificate on Windows'
+    displayName: 'Download certificate on Windows'
 
   - script: |
       set CSC_LINK=$(Agent.TempDirectory)\NORDIC_SIGNING_CERTIFICATE.pfx
       set CSC_KEY_PASSWORD=$(NORDIC_SIGNING_CERTIFICATE_PASSWORD_PFX)
-      npx electron-builder -p never
-    condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'win'))
-    displayName: 'Release on Windows'
+      npx electron-builder -p never --windows nsis:ia32
+    condition: and(
+      ne(variables['Build.Reason'], 'PullRequest'),
+      contains(variables['IMAGE_NAME'], 'win'),
+      eq(variables['NODE_ARCH'], '32'))
+    displayName: 'Build electron installer on Windows x86'
```

Comment:
For me at least it makes more sense to say the task builds the electron installer since the it only runs the `npx electron-builder -p never --<architecture>`  command. Would you want me to change the name for the other platforms, or would you like me to change it back?

Reply:
Yes, you are right, it doesn't really release anything, this step is more about building the electron app. So, please also change the name for the other platforms.